.github/workflows/frontend-code.yml

```diff
@@ -0,0 +1,70 @@
+name: Frontend Code
+
+#############################
+# Start the job on all push #
+#############################
```

Comment:
Can get rid of these comments, the yaml itself should be readable to anyone with a basic understanding of actions